### PR TITLE
Job metrics page use new job exporter metrics

### PR DESCRIPTION
**Pull request by YanjieGao**
Job metrics page use new job exporter metrics & query
Page render doesn't change. Please ref https://github.com/Microsoft/pai/pull/387 pages image.
For example task view page
image

### CODE REVIEW

File: service-deployment/bootstrap/grafana/grafana-configuration/pai-jobview-dashboard.json.template [hunk 1]
```diff
@@ -281,23 +281,23 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "avg by (container_label_PAI_JOB_NAME) (rate(container_network_receive_bytes_total{container_label_PAI_JOB_NAME=~\"$job\"}[5m])) ",
```
Comment (sterowang) [inline service-deployment/bootstrap/grafana/grafana-configuration/pai-jobview-dashboard.json.template]: >[5m] [](start = 144, length = 4)

Why removes 5m time interval config?

Comment (YanjieGao) [reply service-deployment/bootstrap/grafana/grafana-configuration/pai-jobview-dashboard.json.template]: Once use rate function need [5m] like interval. Currently don't use that

**Comment by fanyangCS** [comment]
can you include a screenshot?

**Comment by YanjieGao** [comment]
No change with previous job metrics view, will attach later

**Pull request closed without merging**